packages/tablev2/Column.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from "react";
+import { Sorter } from "./Util";
+
+export type ColumnProps<A, B = string> = {
+  /**
+   * An id given to that column. Needed for persistence.
+   */
+  id: B;
+  /**
+   * Renderer for the cells of that column.
+   */
+  render: (a: A) => React.ReactNode;
```

Comment:
I think we should rename this to `renderer`. It could be confused with [render props](https://reactjs.org/docs/render-props.html)

Reply:
I think this makes sense.